python/eos/plot/plotter.py

```diff
@@ -1961,7 +1961,9 @@ def plot_contents(self):
 
 
     def plot(self):
-        """Produces the plot"""
+        """Produces the plot
+
+        returns fig, ax from matplotlib.pyplot.subplots()"""
```

Comment:
Can we please use the Sphinx syntax for documenting return values?

Reply:
Thanks for the suggestion, I changed it and tested it in sphinx.